### PR TITLE
geckodriver 0.29.0

### PR DESCRIPTION
**Pull request by bollwyvl**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by bollwyvl** [comment]
@conda-forge-admin please rerender

**Comment by bollwyvl** [comment]
@conda-forge/core thoughts about https://github.com/conda-forge/geckodriver-feedstock/pull/17#issuecomment-763292303?

<details>
<summary>My local rerender error...</summary>
<pre>
INFO:conda_smithy.configure_feedstock:Downloading conda-forge-pinning-2021.01.19.11.36.16
INFO:conda_smithy.configure_feedstock:Extracting conda-forge-pinning to /tmp/tmpxo1uo7j2
Setting build platform. This is only useful when pretending to be on another platform, such as for rendering necessary dependencies on a non-native platform. I trust that you know what you're doing.
WARNING:conda_build.config:Setting build platform. This is only useful when pretending to be on another platform, such as for rendering necessary dependencies on a non-native platform. I trust that you know what you're doing.
Setting build arch. This is only useful when pretending to be on another arch, such as for rendering necessary dependencies on a non-native arch. I trust that you know what you're doing.
WARNING:conda_build.config:Setting build arch. This is only useful when pretending to be on another arch, such as for rendering necessary dependencies on a non-native arch. I trust that you know what you're doing.
No numpy version specified in conda_build_config.yaml.  Falling back to default numpy value of 1.11
WARNING:conda_build.metadata:No numpy version specified in conda_build_config.yaml.  Falling back to default numpy value of 1.11
Adding in variants from internal_defaults
INFO:conda_build.variants:Adding in variants from internal_defaults
Adding in variants from /tmp/tmpxo1uo7j2/conda_build_config.yaml
INFO:conda_build.variants:Adding in variants from /tmp/tmpxo1uo7j2/conda_build_config.yaml
Adding in variants from ~/geckodriver-feedstock/recipe/conda_build_config.yaml
INFO:conda_build.variants:Adding in variants from ~/geckodriver-feedstock/recipe/conda_build_config.yaml
Traceback (most recent call last):
  File "~/mc3/bin/conda-smithy", line 10, in <module>
    sys.exit(main())
  File "~/mc3/lib/python3.7/site-packages/conda_smithy/cli.py", line 613, in main
    args.subcommand_func(args)
  File "~/mc3/lib/python3.7/site-packages/conda_smithy/cli.py", line 419, in __call__
    self._call(args, tmpdir)
  File "~/mc3/lib/python3.7/site-packages/conda_smithy/cli.py", line 430, in _call
    temporary_directory=temporary_directory,
  File "~/mc3/lib/python3.7/site-packages/conda_smithy/configure_feedstock.py", line 1953, in main
    render_azure(env, config, forge_dir, return_metadata=True)
  File "~/mc3/lib/python3.7/site-packages/conda_smithy/configure_feedstock.py", line 1221, in render_azure
    return_metadata=return_metadata,
  File "~/mc3/lib/python3.7/site-packages/conda_smithy/configure_feedstock.py", line 608, in _render_ci_provider
    os.path.join(forge_dir, forge_config["recipe_dir"]), config=config
  File "~/mc3/lib/python3.7/site-packages/conda_build/variants.py", line 596, in get_package_combined_spec
    combined_spec = combine_specs(specs, log_output=config.verbose)
  File "~/mc3/lib/python3.7/site-packages/conda_build/variants.py", line 323, in combine_specs
    log_output=log_output)
  File "~/mc3/lib/python3.7/site-packages/conda_build/variants.py", line 298, in _combine_spec_dictionaries
    spec_source))
ValueError: variant config in ~/geckodriver-feedstock/recipe/conda_build_config.yaml is ambiguous because it does not fully implement all zipped keys, or specifies a subspace that is not fully implemented.
</pre>
</details>

<details>
<summary>... with some versions</summary>
<pre>
conda                     4.9.2            py37h89c1867_0    conda-forge
conda-build               3.20.5           py37he5f6b98_0    conda-forge
conda-forge-pinning       2021.01.19.11.36.16      hd8ed1ab_0    conda-forge
conda-lock                0.8.0              pyhd8ed1ab_0    conda-forge
conda-package-handling    1.6.0            py37h7b6447c_0    https://repo.anaconda.com/pkgs/main
conda-smithy              3.8.6              pyhd8ed1ab_0    conda-forge
conda-suggest             0.1.1              pyh9f0ad1d_0    conda-forge
conda-suggest-conda-forge 2020.11.25           ha770c72_0    conda-forge
conda-verify              3.1.1                 py37_1000    conda-forge
</pre>
</details>

**Comment by chrisburr** [comment]
@conda-forge-admin please rerender

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -1,3 +1,5 @@
+cudnn:                                    # [linux64]
```
Comment (bollwyvl) [inline recipe/conda_build_config.yaml:1]: yep... wouldn't have guessed that... also the fact there's the strings `None` and `undefined` within spitting distance of each other terrifies me...

Comment (chrisburr) [reply recipe/conda_build_config.yaml:1]: Yeah the pinnings are fun... 😄

I found it by comparing `recipe/conda-build-config.yaml` with the zipped keys in the global pinnings (more specifically, [here](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/d8507e01df35678a41ad432e9dd3278f73a689e0/recipe/conda_build_config.yaml#L146-L150)).

I've made a PR to the bot to try and improve the error message a little (https://github.com/conda-forge/webservices-dispatch-action/pull/9). If there is anything that you think would help make it clearer let me know.

Comment (bollwyvl) [reply recipe/conda_build_config.yaml:1]: Will comment over there!

**Pull request merged**